### ISSUE TITLE
All meanings of a word on the same card

### ISSUE THREAD

**Issue by jagr21** [damaru2/ankigenbot#16]
Would it be possible to have an option for all meanings of a word to go on the same card, instead of splitting each definition on a new card?

Thank you for creating this amazing tool.

**Comment by damaru2** [comment]
Hi, all three of the things you have written on issues are possible. Although this one in particular is not ideal for several reasons, the main two are:

1. From the interface and implementation point of view, one would need to send another message with a button with all of the information or something like that, which is intrusive for people that do not want this feature.

2. This goes against the **minimum information principle** that is strongly recommended when creating new cards (you can google this principle for anki). 


If I find the time, I will program the other two, but I think this one goes against anki's philosophy.

**Comment by jagr21** [comment]
Ok, I understand. About point 2, I am aware that it is not the ideal practice, however, I got used to doing it this way from the beginning and it has worked quite well for me over time. I think sometimes seeing all the definitions together helps to reinforce the general meaning of a word (I don't know if I'm explaining it well, but I could use the word "hinge" as an example, which I recently added through the bot: the word has two definitions, but they are closely related; with the first one you understand the second one much better).

But I fully understand that it is not ideal to implement. Maybe just as an advanced option? Anyway, I can't ask for more, and it's something I can solve by editing the cards manually later. Thanks for the quick answer :)
